src/01-forEach.js

```diff
@@ -24,13 +26,21 @@ function printAllSongTitles(songs) {}
  *  //> "Up by Sebastian Kamae"
  *  //> ...
  */
-function printSongDetails(songs) {}
+function printSongDetails(songs) {
+  songs.forEach((song) => {console.log(`${song.title} by ${song.artist}`);})
+}
 
 /**
  * Logs out all of the song titles which have a runtime over three minutes.
  * @param {Object[]} songs - An array of songs. See the song data for more.
  */
-function printSongTitlesOverThreeMinutes(songs) {}
+function printSongTitlesOverThreeMinuqtes(songs) {
```

Comment:
There's a typo here that prevents tests for running for `printSongTitlesOverThreeMinutes` (there's an extra q in the function here)